### DISCUSSION TITLE
TP4 - Ejercicio 4 - Division Lenta II - error en el puntero

### DISCUSSION THREAD

**Discussion by JuannHerna** [INGCOM-UNRN-P1/cursada-2023#137] [Preguntas y respuestas] [answered]
Buenas, tengo una consulta, me salta un error en la pasada de parámetros, no entiendo porque me pide doble `**`.
Así está definido en el `main`.
```C
    int resto;
    int* resultres = &resto;
    int cociente;
    int* Rescociente = &cociente;
```
y cuando le asigno doble `**`, me dice que la asignación está mal
![errorejercicio4](https://github.com/INGCOM-UNRN-P1/cursada-2023/assets/105023549/8be59ce6-11d5-4072-b46d-ffc3fcf95b14)

**Comment by martinvilu** [comment, marked as answer]
(Los argumentos van en minúsculas)

Al momento de llamar esa función, esto no requiere de un puntero, simplemente una variable; se usa igual que `scanf`. Que usamos _sin saber_ que era un puntero.

**Comment by JuannHerna** [comment]
aa okey gracias, ahi solucione.

**Reply by martinvilu** [reply]
![capisch](https://github.com/INGCOM-UNRN-P1/cursada-2023/assets/56625/3e0c71e2-4775-42e6-b565-483dcead3582)
